test/interpreter/function-npv.spec.ts

```diff
@@ -0,0 +1,57 @@
+import {ErrorType, HyperFormula} from '../../src'
+import {ErrorMessage} from '../../src/error-message'
+import {adr, detailedError} from '../testUtils'
+
+describe('Function NPV', () => {
+  it('should return #NA! error with the wrong number of arguments', () => {
+    const engine = HyperFormula.buildFromArray([
+      ['=NPV(1)'],
+    ])
+
+    expect(engine.getCellValue(adr('A1'))).toEqualError(detailedError(ErrorType.NA, ErrorMessage.WrongArgNumber))
+  })
+
+  it('should return correct value', () => {
+    const engine = HyperFormula.buildFromArray([
+      ['=NPV(1, B1:C1)', 1, 'abcd'],
+      ['=NPV(1, B2:C2)', true, 1],
+      ['=NPV(1, TRUE(), 1)'],
```

Comment:
Excel documentation says:

> Arguments that are empty cells, logical values, or text representations of numbers, error values, or text that cannot be translated into numbers are ignored.
>
> If an argument is an array or reference, only numbers in that array or reference are counted. Empty cells, logical values, text, or error values in the array or reference are ignored.

But values returned are as in Excel on Mac. Google Spreadsheet has a different opinion:
![Screenshot 2020-10-14 at 19 30 47](https://user-images.githubusercontent.com/4718998/96024749-74875500-0e54-11eb-8164-2267de445965.png)

Reply:
I would split into two scenarios then
```
it('should ignore logical and text values', () => {
      ['=NPV(1, B1:C1)', 1, 'abcd'],
      ['=NPV(1, B2:C2)', true, 1],
```

```
it('should be compatible with Excel ?', () => {
      ['=NPV(1, TRUE(), 1)'],
```

Should we add HF.NPV that will calculate this correctly or move this implementation to MS.NPV and implement this correctly as ODFF and GS does?

Reply:
I think we might mark the inconsistency between implementations. Marking every function where we found it is ok. Implementing two versions of each might be too much (for now).